src/controllers/sdex/index.ts

```diff
@@ -20,6 +24,13 @@ const querySchema = Joi.object({
   poolIdx: Joi.number().required(),
 });
 
+const swapHistoryQuerySchema = Joi.object({
+  user: Joi.string().required(),
+  chainId: Joi.string().required(),
+  page: Joi.number().integer().min(1).default(1),
+  limit: Joi.number().integer().min(1).max(100).default(10),
```

Comment:
limit is already validated by `validatePaginatedRequest`, it also has `cursor` - which we use instead `page`, because it's more performant for tables with a lot of data.
`chainId` is fixed in req.network.chainId in v2 endpoints.

```suggestion
  user: Joi.string().required(),
```

Reply:
got it, now its clear